paymentsheet/src/main/java/com/stripe/android/link/ui/updatecard/UpdateCardScreenState.kt

```diff
@@ -1,19 +1,11 @@
 package com.stripe.android.link.ui.updatecard
 
 import androidx.compose.runtime.Immutable
+import com.stripe.android.model.ConsumerPaymentDetails
+import com.stripe.android.paymentsheet.CardUpdateParams
 
 @Immutable
 internal data class UpdateCardScreenState(
-    val paymentDetailsId: String,
-) {
-
-    internal companion object {
-        fun create(
-            paymentDetailsId: String
-        ): UpdateCardScreenState {
-            return UpdateCardScreenState(
-                paymentDetailsId = paymentDetailsId
-            )
-        }
-    }
-}
+    val paymentDetails: ConsumerPaymentDetails.Card? = null,
```

Comment:
Do we need the full payment details in this state?

Reply:
not really - updated to just save the id + isDefault. https://github.com/stripe/stripe-android/pull/10715/commits/39a1b35696357c46c02106d9ea975fea48bab141